sbin/build-redisjson

```diff
@@ -36,21 +36,22 @@ fi
 if [[ -z $BINROOT ]]; then
 	export BINROOT=$ROOT/bin/$($READIES/bin/platform -t)
 fi
-TARGET_DIR=$BINROOT/RedisJSON
+
+BRANCH=${BRANCH:-master}
+TARGET_DIR=$BINROOT/RedisJSON/$BRANCH
 echo Building into $TARGET_DIR ...
 
 if [[ $FORCE == 1 ]]; then
 	runn rm -rf $TARGET_DIR
 fi
 
-BRANCH=${BRANCH:-master}
-
 $OP cd $ROOT/deps
 if [[ ! -d RedisJSON ]]; then
 	runn git clone --quiet --recursive https://github.com/RedisJSON/RedisJSON.git
 fi
 
 $OP cd RedisJSON
+runn git fetch
```

Comment:
Should be after `git checkout $BRANCH`?

Reply:
I put it before because to extract the branch if it is new